resources/views/components/newsletter.blade.php

```diff
@@ -1,12 +1,11 @@
-{{-- To do: this causes a blank page on the register layout --}}
+@props(['id' => uniqId('newsletter-')])
 
-{{-- @props(['id' => uniqId('newsletter-'), 'vModel'])
 <x-rapidez-ct::card.inactive>
     <x-rapidez-ct::title.lg class="mb-5">
         @lang('Newsletter')
     </x-rapidez-ct::title.lg>
 
-    @if (!isset($vModel))
+    @if (!$attributes->has('v-model'))
         <graphql-mutation
             v-cloak
             query="mutation customer ($is_subscribed: Boolean!) { updateCustomerV2(input: { is_subscribed: $is_subscribed }) { customer { is_subscribed } } }"
```

Comment:
mutations that have different code should have a different name in order to prevent errors by a cached query being excecuted.